client/src/pages/members/Members.js

```diff
@@ -0,0 +1,227 @@
+import {useEffect, useState} from 'react';
+import axios from "axios";
+import TableRow from '../../components/tableRow/TableRow.js';
+import {Edit2} from 'react-feather/';
+import addMemberIcon from '../../assets/add.svg';
+import removeMemberIcon from '../../assets/trashCan.svg';
+import enterEditingIcon from '../../assets/edit.svg';
+import exitEditingIcon from '../../assets/checkmark.svg';
+import './Members.css';
+
+function Members() {
+  const [editingMode, setEditingMode] = useState(false);
+  const [isAddingMember, setIsAddingMember] = useState(false);
+
+	const [proposals, setProposals] = useState();
+  const [members, setMembers] = useState([]);
+
+  /* REACT STATES FOR EDIT/ADD/REMOVE BUTTONS */
+  var addIconDefault = 'addMemberIconContainer';
+  var removeIconDefault = 'removeMembersIconContainer';
+  var enterEditingIconDefault = 'enterEditingIconContainer';
+  var exitEditingIconDefault = 'exitEditingIconContainer';
+
+  const [addIconClassName, setAddIconClassName] = useState(addIconDefault + ' hide');
+  const [removeIconClassName, setRemoveIconClassName] = useState(removeIconDefault + ' hide');
+  const [enterEditingIconClassName, setEnterEditingIconClassName] = useState(enterEditingIconDefault);
+  const [exitEditingIconClassName, setExitEditingIconClassName] = useState(exitEditingIconDefault + ' hide');
+
+  /** Handles clicking the pencil icon to start editing members */
+  function enterEditingMode() {
+    setAddIconClassName(addIconDefault);
+    setRemoveIconClassName(removeIconDefault);
+    setEnterEditingIconClassName(enterEditingIconDefault + ' hide');
+    setExitEditingIconClassName(exitEditingIconDefault);
+    setEditingMode(true);
+  }
+
+  /** Handles clicking the checkmark icon to exit member editing mode */
+  function exitEditingMode() {
+    setAddIconClassName(addIconDefault + ' hide');
+    setRemoveIconClassName(removeIconDefault + ' hide');
+    setEnterEditingIconClassName(enterEditingIconDefault);
+    setExitEditingIconClassName(exitEditingIconDefault + ' hide');
+    setEditingMode(false);
+    setIsAddingMember(false);
+  }
+
+  /** Handles clicking the plus icon to add a new member */
+  function addMember() {
+    if (!isAddingMember) {
+      setIsAddingMember(true);
+    } else {
+      confirmAddMember();
+    }
+  }
+
+  async function confirmAddMember() {
+      var nameTextbox = document.getElementById('addMemberNameTB');
+      var emailTextbox = document.getElementById('addMemberEmailTB');
+      var roleTextbox = document.getElementById('addMemberRoleTB');
+      var privilegeDropdown = document.getElementById('privilegeDropdown');
```

Comment:
Ok *technically* this is bad React convention. It is better in React to use controlled or uncontrolled components instead of document.getElementById (https://www.javascriptstuff.com/use-refs-not-ids/)

Controlled components; https://reactjs.org/docs/forms.html
Uncontrolled components: https://reactjs.org/docs/uncontrolled-components.html

Controlled components are sometimes tedious with all the additional work, so refs (uncontrolled components) might be the way to go.